### PR TITLE
ENH centos7 announcement

### PR DESCRIPTION
**Pull request by beckermr**

### CODE REVIEW

**Comment by beckermr** [comment]
@conda-forge/core Are we ok with the language here? 

I wrote that end of January at the earliest is when we will move to CentOS 7. This gives people the holidays plus another month or so to figure out their stuff.

**Review by kkraus14 — APPROVED** [review]

**Review by mwcraig — APPROVED** [review]

**Review by mbargull — APPROVED** [review]
Very nicely worded, thanks a lot!

File: src/user/announcements.rst [hunk 1]
```diff
@@ -8,6 +8,21 @@ Our announcements are published to an RSS feed `here <https://conda-forge.org/do
 2020
 ----
 
+:2020-12-16: Moving to CentOS 7 and CentOS 6 End-of-Life
+
+    ``conda-forge``'s compiler stack uses repackaged libraries from CentOS 6
+    to supply certain libraries, notably ``glibc`` when building recipes. We currently
+    default to using CentOS 6 with the ``glibc`` 2.12 ABI. However, CentOS6 reached
```
Comment (beckermr) [inline src/user/announcements.rst:15]:
```suggestion
    default to using CentOS 6 with the ``glibc`` 2.12 ABI. However, CentOS 6 reached
```

File: src/user/announcements.rst [hunk 2]
```diff
@@ -8,6 +8,21 @@ Our announcements are published to an RSS feed `here <https://conda-forge.org/do
...
+    end-of-life in November 2020 and increasingly software packages require at
+    least CentOS 7 with the ``glibc`` 2.17 ABI. We also realize that due to recent
+    events some communities that may have been planning to skip CentOS 7
```
Comment (beckermr) [inline src/user/announcements.rst:18]:
```suggestion
    events, some communities that may have been planning to skip CentOS 7
```

**Pull request merged**